.github/workflows/reusable_fast.yml

```diff
@@ -1,76 +1,58 @@
-# Fast builds
```

Comment:
why remove this comment? I would even extend it so we know that linux part is done in dockers, Windows is not.